features/manage_response_sets.feature

```diff
@@ -47,7 +48,7 @@ Feature: Manage Response Sets
     And I click on the option to Extend the Response Set with the name "Gender Full"
     And I fill in the "Name" field with "Gender Partial"
     And I fill in the "Description" field with "M / F / O"
-    And I click on the "Create Response set" button
```

Comment:
This snippet is deceiving, it is actually removing that line from:
```
  Scenario: Extend Response Set
    Given I have a Response Set with the name "Gender Full"
    And I am logged in as test_author@gmail.com
    When I go to the list of Response Sets
    And I click on the option to Extend the Response Set with the name "Gender Full"
    And I fill in the "Name" field with "Gender Partial"
    And I fill in the "Description" field with "M / F / O"
    And I click on the "Extend Response Set" button
    Then I should see "Response set was successfully created."
    And I should see "Gender Partial"
    And I should see "M / F / O"
``` 
aka just edited the button name.  Create new gets tested in lines 56+ of this file.